WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -90,6 +90,11 @@
 
         // list of last time a notification has been created for a draft
         PENDING_DRAFTS_NOTIFICATION_LAST_NOTIFICATION_DATES,
+
+        // smart toast counters
+        SMART_TOAST_PHOTO_PICKER_LONG_PRESS_COUNTER,
+        SMART_TOAST_WP_MEDIA_BROWSER_LONG_PRESS_COUNTER,
+        SMART_TOAST_COMMENTS_LONG_PRESS_COUNTER
```

Comment:
Shouldn't these be `UndeletablePrefKey`s? `DeletablePrefKey`s are reset on WordPress.com logout, so we'll be showing these again after the user logs out and back in. (This is how we're handling the rest of our 'promo'-type keys, too.)